templating/twig_extension_repository.rst

```diff
@@ -40,7 +40,7 @@ Then, define the extension class as a service and tag it with the
 
         # app/config/services.yml
         services:
-            twig_extension.intl:
+            twig_extensions.intl:
```

Comment:
Now that you mention it... For example, I've come across [this one](https://github.com/rdohms/dms-twig-extension-bundle/blob/master/README.md).

Would you guys recommend using a 3rd party bundle instead of providing instructions?

Note I am not affiliated with that bundle in any way and cannot endorse it.